### ISSUE TITLE
Update? On-chain turn based game.

### ISSUE THREAD

**Issue by rayhvh** [ise-ethereum/on-chain-chess#212]
Hello.

For my studies I am trying to build a turn-based on chain game just like this. including using websockets to reduce costs. so movefromstate.
However I found this project to be inactive and I find the code overwhelming. I can't get it to run.

Is there any way you can provide me with a stripped down way / the most basic way of having turn based game with minimal UI and "rules"? 
Thanks so much for reading.

**Comment by ljrahn** [comment]
Agreed. Is there ever any plan to update this? Code base is old and hard to get running

**Comment by graup** [comment]
Sorry, no plans to update this. It's an old project and the team doesn't exist anymore. All I can offer is my write-up about lessons learned: https://medium.com/@graycoding/lessons-learned-from-making-a-chess-game-for-ethereum-6917c01178b6
